### DISCUSSION TITLE
Mouse Keys remapping works on PC1 but not on PC2—previously functional, now broken.

### DISCUSSION THREAD

**Discussion by DonVegaaR** [input-leap/input-leap#2126] [Q&A]
I remapped the mouse scroll to the side buttons on PC1 (InputLeap server, Windows) because the scroll wheel is broken. This used to work on PC2 (client, Ubuntu), but after rebooting, scrolling with the side buttons no longer works.

Any idea how to fix this? The client should receive input as if the devices were physically attached, but I don’t know why it worked before and doesn’t now

I'm new to this and have little knowledge of coding, but if someone could point me in the right direction to test input or understand how InputLeap manages the keys, I would be grateful

**Comment by shymega** [comment]
Please open a bug.
